kili/mutations/user/__init__.py

```diff
@@ -110,11 +137,23 @@ def reset_password(self, email: str):
         result = self.auth.client.execute(GQL_RESET_PASSWORD, variables)
         return format_result('data', result)
 
+    @deprecate(
+        """
+        The parameter "name" is deprecated since: 11/01/2022.
+        It will be removed after: 21/02/2022.
+        Parameters "firstname" and "lastname" have to be used instead.
+            > kili.update_properties_in_user(
+                email='john.miller@kili-technology.com',
+                firstname='John',
+                lastname='Miller')
+        """)
     @Compatible(['v1', 'v2'])
     @typechecked
     def update_properties_in_user(self,
                                   email: str,
                                   name: Optional[str] = None,
+                                  firstname: Optional[str] = None,
```

Comment:
Same

Reply:
Here they are optional arguments, so it's ok